sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_retrieval/_retrieval.py

```diff
@@ -93,7 +94,7 @@ async def __call__(self, *, conversation, **kwargs):
         }
 
 
-class RetrievalChatEvaluator:
+class RetrievalScoreEvaluator:
```

Comment:
to align with other quality evaluators, why not just call it RetrievalEvaluator?

Reply:
Minsoo specifically asked for this name instead of RetrievalEvaluator, but I'll check with her.